pages/job-category/index.js

```diff
@@ -0,0 +1,157 @@
+import Link from "next/link";
+import { styled } from "styled-components";
+import JobList from "../components/JobList";
+import { v4 as uuidv4 } from "uuid";
+import tailwindConfig from "@/tailwind.config";
+import postcssConfig from "@/postcss.config";
+import Image from "next/image";
+
+const StyledArticle = styled.article`
+  border-radius: 20px;
+  border: 1px solid lightgrey;
+  margin: 5px;
+  padding: 10px;
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 20px;
+`;
+
+const StyledHeading = styled.h1`
+  text-align: center;
+  color: white;
+`;
+const StyledSpan = styled.span`
+  border-radius: 20px;
+  border: 1px lightgrey;
+  box-shadow: 1px 1px 2px lightgrey;
+  background-color: #c8c8c8;
+  margin: 5px;
+  padding: 10px;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+  font-size: 20px;
+`;
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10%;
+  webkit-box-pack: center;
+  flex-direction: row;
+  justify-content: center;
+  background-color: white;
+`;
+
+const StyledDiv1 = styled.div`
+  display: flex;
+  text-align: center;
+  flex-align: center;
+  postion: absolute;
+  left: 30px;
+`;
+
+const StyledP = styled.p`
+  color: white;
```

Comment:
I think we can remove this, and put the color property on the element that wraps the `heading` and the `p`. I.e. to what is currently called `StyledDiv1`. It will then be inherited by the children.